ngraph/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM tensorflow/tensorflow:1.12.0
+
+RUN apt-get update && apt-get install -y vim
+
+# add google cloud storage
+RUN pip --no-cache-dir install \
+        google-cloud-storage \
+        && \
+    python -m ipykernel.kernelspec
+
+RUN  pip install -U ngraph-tensorflow-bridge && \
+python -c "import tensorflow as tf; print('TensorFlow version: r',tf.__version__);import ngraph_bridge; print(ngraph_bridge.__version__)"
```

Comment:
Why are you printing out the ngraph version as part of building the Docker image? 

Would it be better to pin the version of the ngraph library installed in the image
e.g
```
pip install -U ngraph--tensorflow-bridge==X.X.X
```

Reply:
Yes, I'll include by version, thanks